tests/python/relax/test_tvmscript_parser.py

```diff
@@ -254,7 +256,7 @@ def main(x: R.Tensor((10, 20), "float32")) -> R.Tensor((10, 20), dtype="float32"
 
     bb = relax.BlockBuilder()
     x = relax.Var("x", relax.TensorStructInfo([10, 20], "float32"))
-    with bb.function("main", [x]):
+    with bb.function("main", [x], {"global_symbol": "main"}):
```

Comment:
how about we also have private=False flag here(default to False)

Reply:
`private=False` in the parser (which is the default) means that there's a global symbol in the attributes.

Reply:
Do you mean the block builder should also have a private flag in the constructor?

Reply:
Right, this way it is closer to what function provide

Reply:
I've implemented that, good idea :+1: